services/shared/canteen.ts

```diff
@@ -48,5 +48,5 @@ export interface Booking extends GenericInterface  {
 	label: string,
 	canBook: boolean,
 	booked: boolean,
-	ref?: TurboBookingDay
+	ref?: TurboBookingDay | import("alise-api").BookingDay
```

Comment:
```suggestion
	ref?: TurboBookingDay | BookingDay
```
+ import Alise au top du fichier

Reply:
c'est fait !